dracut/modules.d/90kiwi-dump/kiwi-dump-image.sh

```diff
@@ -354,17 +354,17 @@ function get_remote_image_source_files {
 
     image_uri=$(getarg rd.kiwi.install.image=)
     image_md5_uri=$(
-        echo "${image_uri}" | awk '{ gsub("\\.xz",".md5", $1); print $1 }'
+      echo "${image_uri}" | awk '{ gsub("\\.xz",".md5", $1); gsub("dolly:","tftp:", $1); print $1 }'
     )
     image_initrd_uri=$(
-        echo "${image_uri}" | awk '{ gsub("\\.xz",".initrd", $1); print $1 }'
+      echo "${image_uri}" | awk '{ gsub("\\.xz",".initrd", $1); gsub("dolly:","tftp:", $1); print $1 }'
     )
     image_kernel_uri=$(
-        echo "${image_uri}" | awk '{ gsub("\\.xz",".kernel", $1); print $1 }'
+      echo "${image_uri}" | awk '{ gsub("\\.xz",".kernel", $1); gsub("dolly:","tftp:", $1); print $1 }'
     )
     image_config_uri=$(
         echo "${image_uri}" | \
-        awk '{ gsub("\\.xz",".config.bootoptions", $1); print $1 }'
+        awk '{ gsub("\\.xz",".config.bootoptions", $1); gsub("dolly:","tftp:", $1); print $1 }'
```

Comment:
If I understand the concept right you want to use ```dolly:``` only for the system image but for the "metadata" we stick with tftp. If so I prefer if the replacement will be done only once and also explained as it's not obvious why it will be done

```bash
# make sure the protocol type is tftp for metadata files. There is no need for
# complex protocol types on small files and for standard PXE boot operations
# only tftp can be guaranteed
image_uri=echo "${image_uri}" | awk '{ gsub("dolly:","tftp:", $1); print $1 }'

...keep the rest of the code untouched
```

Reply:
don't see that this was resolved

Reply:
Nope, we can't manipulate the `$image_uri` variable as the 'dolly' keyword inside it is used in another function to detect if we have to use dolly or curl.

Reply:
Oh I see, yes we return ```echo "${image_uri}|${image_md5}"``` sorry I overlooked that part.

> but I will fix the spaces and resubmit after the weekend.

Thanks